### PR TITLE
Unit tests for messaging buttons

### PR DESCRIPTION
**Pull request by bshyong**
resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/795

### CODE REVIEW

File: test/messaging/components/buttons/ButtonDelete.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  it('should handle click action properly', () => {
+    const onClickHandler = sinon.spy();
+    const buttonDelete = ReactTestUtils.renderIntoDocument(
+      <ButtonDelete
+          onClickHandler={onClickHandler}/>
+    );
+
+    buttonDelete.handleClick();
+    expect(onClickHandler.called).to.be.true;
```
Comment (U-DON) [inline test/messaging/components/buttons/ButtonDelete.unit.spec.jsx]: The other buttons have click handlers too, but why is this (and print) the only one that's tested? Is it based on the test coverage report?

Comment (U-DON) [inline test/messaging/components/buttons/ButtonDelete.unit.spec.jsx]: Also, `onClickHandler` seems redundant... Probably can just rename to `onClick` like the others for consistency.

File: src/js/messaging/components/buttons/ButtonPrint.jsx [hunk 2]
```diff
@@ -23,6 +24,7 @@ class ButtonPrint extends React.Component {
 }
 
 ButtonPrint.propTypes = {
+  onClick: React.PropTypes.func,
 };
 
 export default ButtonPrint;
```
Comment (U-DON) [inline src/js/messaging/components/buttons/ButtonPrint.jsx]: `ButtonPrint` actually isn't being used at all right now, but I don't know if a print feature is going to be added in the future.

**Review by U-DON — APPROVED** [review]
LGTM with some comments.

File: test/messaging/components/buttons/ButtonDelete.unit.spec.jsx (hunk 1 above)
Comment (bshyong) [inline test/messaging/components/buttons/ButtonDelete.unit.spec.jsx]: The other buttons don't have an internal `handleClick` function; they just pass down the onClick from the parent directly into their child button components, so there is nothing really to test there

**Pull request merged**